Java/syntax_test_java.java

```diff
@@ -88,15 +92,15 @@ public static void main(String... args) {
             System.out.println(i);
         }
     }
-
+//  ^ meta.method.java meta.method.body.java punctuation.definition.method.end.java
```

Comment:
This should be testing for `punctuation.section.block.end.java`

Reply:
Done